cmd/maestro/maestro.go

```diff
@@ -89,9 +89,8 @@ The options are:
 	go baton.Run()
 	m := http.NewServeMux()
 	m.Handle("/baton/", http.StripPrefix("/baton", baton.Handler()))
+	m.Handle("/webhooks/", http.StripPrefix("/webhooks", baton.WebhookHandler()))
```

Comment:
Might it make sense to register this in https://github.com/hackedu/maestro/blob/master/baton/handler.go#L12?